slack.tex

```diff
@@ -1,7 +1,53 @@
 \subsection{Slack}
 
-Description of the types of communications that happen on Slack
+The data management group uses instant messaging tools to answer questions, 
+share code snippets and errors, and coordinate activities in real-time.  
+Before using instant messaging, we would communicate primarily via e-mail or 
+over the phone. While these do still have their place, being able to have 
+conversations in a team-wide tool has been very beneficial. Discussions happen 
+with more team members participating, instead of just who happened to be on a 
+phone call or an e-mail thread. Questions and concerns have been able to be 
+answered much more quickly.
```

Comment:
Maybe say that we have some channels dedicated to non-work topics to allow friendships and team building to happen outside of the work context. We also have public channels for each site to allow local discussions

Reply:
I was wondering whether to add something about the non-work channels, since I wasn't sure of the "voice" of the paper.  I'm adding this in now.